peerdid.core not included in published package?

I tried using peerdid but am running into the following error on import:

```
.venv/lib/python3.10/site-packages/peerdid/__init__.py:3: in <module>
    from . import core, dids, errors, keys
.venv/lib/python3.10/site-packages/peerdid/dids.py:9: in <module>
    from .core.peer_did_helper import (
E   ModuleNotFoundError: No module named 'peerdid.core'
```

When checking my site-packages directory, I found that only the following files were downloaded from PyPI:
```
dids.py  errors.py  __init__.py  keys.py 
```

Is there perhaps an issue with the setup of the package? Or is my local environment somehow getting corrupted? I wiped my virtual environment and reinstalled dependencies but that resulted in the same error.

cc @chumbert

--- a/peerdid/__init__.py
+++ b/peerdid/__init__.py
@@ -4,6 +4,6 @@ from . import core, dids, errors, keys
 
 from pydid import DID, DIDDocument
 
-__version__ = "0.5.0"
+__version__ = "0.5.1"
 
 __all__ = ["__version__", "core", "errors", "dids", "keys", "DID", "DIDDocument"]

--- a/setup.cfg
+++ b/setup.cfg
@@ -28,7 +28,7 @@ classifiers =
     # Topic ::
 
 [options]
-packages = peerdid
+packages = peerdid,peerdid.core
 include_package_data = true
 python_requires = >= 3.7
 # Dependencies are in setup.py for GitHub's dependency graph.

